samples/LoggingTracer/LoggingTracer/LoggingSpanBuilder.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+using OpenTelemetry.Common;
+using OpenTelemetry.Trace;
+
+namespace LoggingTracer
+{
+    public class LoggingSpanBuilder : ISpanBuilder
+    {
+        private string spanName;
+        private SpanKind spanKind;
+        private ISpan parent;
+        private SpanContext remoteParentSpanContext;
+
+        public LoggingSpanBuilder(string spanName, SpanKind spanKind)
+        {
+            Logger.Log($"SpanBuilder.ctor({spanName})");
+
+            this.spanName = spanName;
+            this.spanKind = spanKind;
+        }
+
+        public LoggingSpanBuilder(string spanName, SpanKind spanKind, ISpan parent) : this(spanName, spanKind)
+        {
+            Logger.Log($"SpanBuilder.ctor({spanName})");
+            this.parent = parent;
+        }
+
+        public LoggingSpanBuilder(string spanName, SpanKind spanKind, SpanContext remoteParentSpanContext) : this(spanName, spanKind)
+        {
+            Logger.Log($"SpanBuilder.ctor({spanName})");
+            this.remoteParentSpanContext = remoteParentSpanContext;
+        }
+
+        public ISpanBuilder SetParentLinks(IEnumerable<ISpan> parentLinks)
+        {
+            Logger.Log($"SpanBuilder.SetParentLinks");
+            return this;
+        }
+
+        public ISpanBuilder SetRecordEvents(bool recordEvents)
+        {
+            Logger.Log($"SpanBuilder.SetRecordEvents");
+            return this;
+        }
+
+        public ISpanBuilder SetSampler(ISampler sampler)
+        {
+            Logger.Log($"SpanBuilder.SetSampler");
+            return this;
+        }
+
+        public IScope StartScopedSpan()
+        {
+            Logger.Log($"SpanBuilder.StartScopedSpan");
+            return new CurrentSpanUtils.LoggingScope(new LoggingSpan(spanName, spanKind));
```

Comment:
you don't need to create a new span every time. One instance should suffice

Reply:
is it?

Reply:
ok. creating the span in .ctor now and re-use that instance.

interesting question though: If I use one SpanBuilder, should I be able to create multiple spans from that (multiple calls to StartScopedSpans)?